CONTRIBUTORS

```diff
@@ -42,13 +55,21 @@ SIG-archinfra:
 
 
 SIG-operators:
+    - ashbhandare
     - bowenbao
     - daquexian
     - garymm
     - hariharans29
+    - hwangdeyu
+    - jiafatom
+    - liuyu21
     - matteosal
+    - neginraoof
```

Comment:
I don't expect them to continue to contribute

Reply:
updated to not adding neginraoof.